string computer_path
string[] camera_paths
---
bool success
